source/system/Mutex.ooc

```diff
@@ -56,8 +56,6 @@ MutexGlobal: class extends Mutex {
 	lock: override func { This _globalMutex lock() }
 	unlock: override func { This _globalMutex unlock() }
 	free: static func ~all { This _globalMutex free() }
-
-	GlobalCleanup register(|| This free~all())
```

Comment:
Thanks! Putting the line in `Memory.ooc` worked, so I think it might be caused by the order that static memory is initialized.